initializr-service/src/main/resources/application.yml

```diff
@@ -1198,6 +1206,41 @@ initializr:
           id: remote-shell
           description: CRaSH shell integration
           versionRange: "[1.0.0.RELEASE,1.5.0.RC1)"
+    - name: Spring Cloud GCP
+      bom: spring-cloud-gcp
+      versionRange: 1.0.0.M2
+      content:
+        - name: Spring Cloud GCP Starter
+          id: spring-cloud-gcp-starter
+          description: Starter for Spring Cloud GCP
+          groupId: org.springframework.cloud
+          artifactId: spring-cloud-gcp-starter
+          starter: true
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-gcp/docs/1.0.0.M2/reference/htmlsingle/
```

Comment:
We can't hardcode `1.0.0.M2` in there. Also, a dedicated section that explains what we can do would be appreciated. 

It could be a section that summarizes the capabilites of Spring Cloud GCP with links to  each section .What I envision is something that let a user to quickly understand what they can do. Each link would then provide a quickstart guide for the "feature".

Reply:
Did you mean a dedicated section in the refdoc? If so, that's what I intended with that short [intro](https://docs.spring.io/spring-cloud-gcp/docs/1.0.0.M1/reference/htmlsingle/#_introduction).

Unfortunately, I think 1.0.0.M2 is [the best we have](https://docs.spring.io/spring-cloud-gcp/docs/).

If you have ideas on how to improve the refdoc, I'd ask you to file an issue in our repo that we can follow up when we've got the time.
At this point, I wouldn't want to block M2/Initializr support on that refdoc improvement if possible.